### ISSUE TITLE
Too much spam

### ISSUE THREAD

**Issue by Metalhead33** [ars3niy/tdlib-purple#157]
As someone else already mentioned, messages are received from chats that we're not members of _( #150 )_, but for me, it goes even beyond that: from those chats where I am not a member - not even subscribed - I also consistently receive a flurry of image, video, music and document downloads. As such I often get over 250 open dialogs about downloading a file. In many cases, all of those appear instantly, effectively freezing my system _(killing my KDE / X-server, forcing me to reboot my system)_.

Is there a way to disable these file downloads as a whole? Or maybe a solution for #150 , so that we won't receive these unsolicited messages in the first place, let alone the unsolicited flurry of gigabytes worth of data?

Until that, I have to switch back to [the unmaintained telegram pidgin plugin](https://github.com/majn/telegram-purple), which at least doesn't spam me with the messages. Keep up the good work though.

**Comment by bmomjian** [comment]
Yes, this is a serious problem if there are malicious people trying to attack your account.  I installed @BenWiederhake's fork mentioned in #163 and am hopeful that will fix it.  It would be nice to get a new packaged release so manual builds would not be necessary.
